File: programmers/없는 숫자 더하기.py
def solution(numbers):
    answer = 0
    num=[1,2,3,4,5,6,7,8,9]
    visited =[0]*10
    for i in numbers:
        visited[i]=1
    
    for i in range(0,10):
        if visited[i]==0:
            answer+=i
    return answer

File: programmers/Level 1/없는 숫자 더하기.js

//나의 풀이 
function solution(numbers) {
    var answer = 45;
    for (var value of numbers) {
        answer-=value
    }
    return answer;
}

//다른 사람의 풀이
function solution(numbers) {
    return 45 - numbers.reduce((cur, acc) => cur + acc, 0);
}

function solution(numbers) {
    let answer = 0;

    for(let i = 0; i <= 9; i++) {
        if(!numbers.includes(i)) answer += i;
    }

    return answer;
}
